autogluon/utils/tabular/ml/models/tab_transformer/hyperparameters/kwargs.py

```diff
@@ -0,0 +1,43 @@
+#kwargs for TabTransformer model and training
+
+def get_kwargs(**kw):
+    kwargs =   {'batch_size': 512, 
+                'tab_kwargs': {'n_cont_embeddings': 0,
+                               'n_layers': 1,
```

Comment:
Add comments describing what each hyperparameter represents. It would also be helpful if your comments list (semi-)reasonable search-space to consider for each hyperparameter during HPO.